js/src/personal/bryan/Bryan.module.css

```diff
@@ -33,11 +33,13 @@
 }
 
 .container {
-  min-height: 80vh;
+  min-height: 85vh;
   display: flex;
+  justify-content: flex-start;
   flex-direction: column;
 }
 
 .footer {
+  margin-left: -40rem;
```

Comment:
Generally I tend to avoid negative margins. What is this for? Can you do this another way?